### ISSUE TITLE
Caching problem with `xsi:schemaLocation`

### ISSUE THREAD

**Issue by jakubklimek** [redhat-developer/vscode-xml#1027]
I am developing XSDs and valid XML examples and publishing them to the web, so that I can point to them using absolute IRIs in the xsi:schemaLocation.

However, I am getting validation errors based on previous (days old) XSD version, which has been replaced since. It seems like the plugin is using some sort of a cache. My XSDs reference other XSDs and so on, and some of them seem not to update for validation.

My test XML with schema location (note that Chrome cannot handle IRIs in namespaces, Firefox is fine):
https://ofn.gov.cz/společné-části-elektronických-dokumentů/2024-10-04/příklady/fyzická-osoba-ve-vztahu-k-dokumentu.xml

And the error (which is incorrect):
![image](https://github.com/user-attachments/assets/2e9e1083-3ce5-47a0-a05d-df98d75e2ae1)

**Comment by angelozerr** [comment]
Indeed we use cache to have the best performance.

If I remember, if you execute `Revalidate all opened XML files` command, it should evict the cache 

![image](https://github.com/user-attachments/assets/d32071c0-fb54-4dc6-b06e-86d3c334b95c)

**Comment by fbricon** [comment]
yup, see https://github.com/redhat-developer/vscode-xml/pull/343#issuecomment-709120128

**Comment by jakubklimek** [comment]
Yes, this works, thanks. Even though... it would make more sense to somehow communicate this part of the feature to the users, maybe rename the command to "Revalidate all opened XML files and evict cache" or something like this.

**Comment by angelozerr** [comment]
We have documentation which explains that at https://github.com/redhat-developer/vscode-xml/blob/main/docs%2FCommands.md and this documentation is too available inside vscode with the Open XML documentation command

**Comment by jakubklimek** [comment]
OK, thanks.
